velox/exec/HashProbe.cpp

```diff
@@ -799,6 +808,21 @@ int32_t HashProbe::evalFilter(int32_t numRows) {
     }
   } else if (isNullAwareAntiJoin(joinType_)) {
     numPassed = evalFilterForNullAwareAntiJoin(numRows, filterPropagateNulls);
+  } else if (isAntiJoin(joinType_)) {
+    auto addMiss = [&](auto row) {
```

Comment:
Consider to put this in evalFilterForAntiJoin?

Reply:
This is more closer to other join types than null-aware anti so I just inline it here.